isis/src/control/objs/BundleUtilities/BundleObservation.h

```diff
@@ -120,20 +133,23 @@ namespace Isis {
      
       LinearAlgebra::Vector &parameterWeights();
       LinearAlgebra::Vector &parameterCorrections();
-//    LinearAlgebra::Vector &parameterSolution();
       LinearAlgebra::Vector &aprioriSigmas();
       LinearAlgebra::Vector &adjustedSigmas();
       
       const BundleObservationSolveSettingsQsp solveSettings();
 
-//    QStringList serialNumbers();
-
       bool applyParameterCorrections(LinearAlgebra::Vector corrections);
       bool initializeExteriorOrientation();
       void initializeBodyRotation();
       void updateBodyRotation();
 
-      QString formatBundleOutputString(bool errorPropagation, bool imageCSV=false);
```

Comment:
This is API breaking :(((((((((

Reply:
Is it worth it to try to maintain the old API to get this feature into the release?

Reply:
that may be the solution for right now. Maybe just flag it as deprecated?

Reply:
What would be the best way to flag deprecation? And is the idea here to re-implement the monstrous code that was removed...?

Reply:
By re-implement, I just mean copy and paste the old code along side the new code. Flagging as deprecated is as simple as warning when the method gets called. c++14 has special things for deprecated functions, but we only use c++11 right now.